### Publicly available data provide evidence against NR1H3 R415Q causing multiple sclerosis

Eric Vallabh Minikel<sup>1,2,3</sup>

1. Program in Medical and Population Genetics, Broad Institute of MIT and Harvard
2. Analytical and Translational Genetics Unit, Massachusetts General Hospital
3. Program in Biological and Biomedical Sciences, Harvard Medical School

eminikel@broadinstitute.org

Wang et al. report that an NR1H3 variant, rs61731956, encoding an R415Q substitution, causes a novel familial form of multiple sclerosis (MS) (Wang et al., 2016). This claim of pathogenicity rests on imperfect segregation with disease in two pedigrees, and is at odds with publicly available data from the Exome Aggregation Consortium (ExAC) (Exome Aggregation Consortium et al., In press).

ExAC is a reference database of genetic variation among 60,706 ancestrally diverse individuals, assembled from multiple international consortia and made publicly available online (http://exac.broadinstitute.org) since October 2014. By providing population allele frequency information, it has enabled reclassification of over 150 reportedly pathogenic variants as benign (Exome Aggregation Consortium et al., In press), has allowed candidate variants to be ruled in or out as potentially disease-causing (Mencacci et al., 2016; Watts et al., 2015; Weren et al., 2015), has permitted re-evaluation of the interpretability of rare variants in specific disease genes (Walsh, 2016) and estimation of the penetrance of individual rare disease variants (Minikel et al., 2016), and has been used to demonstrate enrichment of pathogenic variants in cases over population controls (Singh et al., 2016). 

As the authors note, the NR1H3 R415Q variant was found in 21 ExAC individuals (“ExAC variant 11:47290147 G / A,” 2014). These 21 individuals are all of non-Finnish European ancestry, with 66,738 non-Finnish European chromosomes having genotype calls for this variant, for an allele frequency of 0.031% in this population -- within the range reported for different European populations by Cotsapas et al. While this genetic variant is not common, it is nevertheless too frequent to be causal for a novel familial form of multiple sclerosis.

Enrichment in cases over controls is one important criterion for establishing the causality of sequence variants for genetic disease (MacArthur et al., 2014; Richards et al., 2015). The authors identified the NR1H3 R415Q variant in one family and subsequently found it in 1 out of 2053 individuals in a multiple sclerosis case series, for an allele frequency of 0.024%, or 0.049% if the proband from the original family is included. The ancestry distribution of this case series is not stated by Wang et al., but the series was collected in Canada and appears to be of predominantly European ancestry (Sadovnick et al., 1998; Traboulsee et al., 2014), enabling a rough comparison to the ExAC non-Finnish Europeans. Even if two individuals with this variant are included in the case series, the variant is not significantly enriched in cases over ExAC population controls (P = .39). 

Variants that cause Mendelian disease, even with incomplete penetrance, should be dramatically enriched in cases over controls. A systematic review found best estimates of lifetime risk of MS to be 0.25% for women and 0.14% for men (Alonso and Hernán, 2008). A 0.25% lifetime risk means that a woman's odds of developing MS are 1:399. Thus, a genetic variant that conferred even 50% lifetime risk (50/50 odds) of developing MS would have an odds ratio of 399. Such a genetic variant would therefore be expected to be enriched to a similar degree – by a few hundred fold – in MS cases compared to controls. Phrased differently, if NR1H3 R415Q has an allele frequency of 0.031% (0.06% of people are heterozygotes) and causes a 50% lifetime risk of MS, then this variant alone would result in 0.03% of the population developing MS. Comparison of this figure to a total of 0.25% of women and 0.14% of men developing MS (Alonso and Hernán, 2008) would imply that NR1H3 R415Q should be found in ~12-21% of all individuals with MS, rather than <0.1% as seen in the reported case series. 

ExAC data do not prove that NR1H3 R415Q lacks any association at all to increased MS risk, but they do put limits on the potential penetrance of this variant. The lifetime risk conferred by disease-associated variants can be estimated using case and control allele frequencies plus an estimate of baseline disease risk (Kirov et al., 2014; Minikel et al., 2016). Assuming 2 alleles in 2053 cases and using the lifetime risk figures noted above, the upper bound of the 95% confidence interval of penetrance for R415Q can be estimated at 2.2% for women and 1.2% for men, indicating that even if this variant is disease-associated, individuals harboring the variant would have a lifetime risk of developing MS no higher than a few percent.

As with any large-scale resource, there are some caveats that users should consider when leveraging ExAC data for variant interpretation, but none of these preclude the application of ExAC in this instance. Individuals in ExAC are not necessarily disease-free – approximately half were ascertained for adult-onset diseases such as type 2 diabetes and schizophrenia – however, none of the ExAC cohorts has been ascertained for MS. 1,675 (2.8%) of individuals in ExAC were ascertained as cases or controls for inflammatory bowel disease (IBD), but the degree of genetic correlation and possible co-morbidity between IBD and MS (Cotsapas et al., 2011; Nielsen et al., 2008; Ramagopalan et al., 2007; Sadovnick et al., 1989) is not sufficient to result in overall enrichment of ExAC for MS cases to any appreciable extent. Despite extensive quality control measures, ExAC still contains some genotyping errors (Rodan et al., 2016), but we estimate a false discovery rate of <0.2% for single nucleotide variants (Exome Aggregation Consortium et al., In press), and raw read visualizations allow users to personally inspect variant quality, revealing that NR1H3 R415Q genotype calls are indeed of high quality. Finally, many benign variants are very rare, so allele frequency data cannot refute all erroneous claims of pathogenicity, and ExAC is therefore no substitute for the observance of strict statistical thresholds, such as LOD ≥ 3.0 for linkage studies (Lander and Kruglyak, 1995), which was not met in the segregation analysis reported by Wang et al. Notwithstanding all of these limitations, ExAC provides a valuable resource for human genetics, and for this variant it provides information sufficient to effectively rule out causality for a familial form of MS.

Comparison of large case-control series, sequenced or genotyped using the same technology, and employing appropriate controls for population stratification, is the best way to evaluate whether candidate disease-causing variants are enriched in cases. Such datasets do not exist for all diseases, however, nor do all investigators have access to such series where they do exist. ExAC data provide a publicly available resource that anyone can use to evaluate the plausibility of putative Mendelian disease variants, and it is critical that authors and reviewers carefully consider such resources, and perform rigorous and quantitative assessments, when evaluating claims of variant pathogenicity.

#### Acknowledgments

*The R source code for calculations described here is publicly available at https://github.com/ericminikel/nr1h3. EVM is supported by the National Institutes of Health under a Ruth L. Kirschstein National Research Service Award (NRSA) NIH Individual Predoctoral Fellowship (F31) (award AI122592-01A1), and thanks Daniel MacArthur for discussions.*

#### References

1.	Alonso, A., Hernán, M.A., 2008. Temporal trends in the incidence of multiple sclerosis: a systematic review. Neurology 71, 129–135. doi:10.1212/01.wnl.0000316802.35974.34
2.	Cotsapas, C., Voight, B.F., Rossin, E., Lage, K., Neale, B.M., Wallace, C., Abecasis, G.R., Barrett, J.C., Behrens, T., Cho, J., De Jager, P.L., Elder, J.T., Graham, R.R., Gregersen, P., Klareskog, L., Siminovitch, K.A., van Heel, D.A., Wijmenga, C., Worthington, J., Todd, J.A., Hafler, D.A., Rich, S.S., Daly, M.J., FOCiS Network of Consortia, 2011. Pervasive sharing of genetic effects in autoimmune disease. PLoS Genet. 7, e1002254. doi:10.1371/journal.pgen.1002254
3.	ExAC variant 11:47290147 G / A [WWW Document], 2014. URL http://exac.broadinstitute.org/variant/11-47290147-G-A (accessed 6.30.16).
4.	Exome Aggregation Consortium, Lek, M., Karczewski, K., Minikel, E., Samocha, K., Banks, E., Fennell, T., O’Donnell-Luria, A., Ware, J., Hill, A., Cummings, B., Tukiainen, T., Birnbaum, D., Kosmicki, J., Duncan, L., Estrada, K., Zhao, F., Zou, J., Pierce-Hoffman, E., Cooper, D., DePristo, M., Do, R., Flannick, J., Fromer, M., Gauthier, L., Goldstein, J., Gupta, N., Howrigan, D., Kiezun, A., Kurki, M., Moonshine, A.L., Natarajan, P., Orozco, L., Peloso, G., Poplin, R., Rivas, M., Ruano-Rubio, V., Ruderfer, D., Shakir, K., Stenson, P., Stevens, C., Thomas, B., Tiao, G., Tusie-Luna, M., Weisburd, B., Won, H.-H., Yu, D., Altshuler, D., Ardissino, D., Boehnke, M., Danesh, J., Roberto, E., Florez, J., Gabriel, S., Getz, G., Hultman, C., Kathiresan, S., Laakso, M., McCarroll, S., McCarthy, M., McGovern, D., McPherson, R., Neale, B., Palotie, A., Purcell, S., Saleheen, D., Scharf, J., Sklar, P., Patrick, S., Tuomilehto, J., Watkins, H., Wilson, J., Daly, M., MacArthur, D., In press. Analysis of protein-coding genetic variation in 60,706 humans. Nature. doi:10.1101/030338
5.	Kirov, G., Rees, E., Walters, J.T.R., Escott-Price, V., Georgieva, L., Richards, A.L., Chambert, K.D., Davies, G., Legge, S.E., Moran, J.L., McCarroll, S.A., O’Donovan, M.C., Owen, M.J., 2014. The penetrance of copy number variations for schizophrenia and developmental delay. Biol. Psychiatry 75, 378–385. doi:10.1016/j.biopsych.2013.07.022
6.	Lander, E., Kruglyak, L., 1995. Genetic dissection of complex traits: guidelines for interpreting and reporting linkage results. Nat. Genet. 11, 241–247. doi:10.1038/ng1195-241
7.	MacArthur, D.G., Manolio, T.A., Dimmock, D.P., Rehm, H.L., Shendure, J., Abecasis, G.R., Adams, D.R., Altman, R.B., Antonarakis, S.E., Ashley, E.A., Barrett, J.C., Biesecker, L.G., Conrad, D.F., Cooper, G.M., Cox, N.J., Daly, M.J., Gerstein, M.B., Goldstein, D.B., Hirschhorn, J.N., Leal, S.M., Pennacchio, L.A., Stamatoyannopoulos, J.A., Sunyaev, S.R., Valle, D., Voight, B.F., Winckler, W., Gunter, C., 2014. Guidelines for investigating causality of sequence variants in human disease. Nature 508, 469–476. doi:10.1038/nature13127
8.	Mencacci, N.E., Kamsteeg, E.-J., Nakashima, K., R’Bibo, L., Lynch, D.S., Balint, B., Willemsen, M.A.A.P., Adams, M.E., Wiethoff, S., Suzuki, K., Davies, C.H., Ng, J., Meyer, E., Veneziano, L., Giunti, P., Hughes, D., Raymond, F.L., Carecchio, M., Zorzi, G., Nardocci, N., Barzaghi, C., Garavaglia, B., Salpietro, V., Hardy, J., Pittman, A.M., Houlden, H., Kurian, M.A., Kimura, H., Vissers, L.E.L.M., Wood, N.W., Bhatia, K.P., 2016. De Novo Mutations in PDE10A Cause Childhood-Onset Chorea with Bilateral Striatal Lesions. Am. J. Hum. Genet. 98, 763–771. doi:10.1016/j.ajhg.2016.02.015
9.	Minikel, E.V., Vallabh, S.M., Lek, M., Estrada, K., Samocha, K.E., Sathirapongsasuti, J.F., McLean, C.Y., Tung, J.Y., Yu, L.P.C., Gambetti, P., Blevins, J., Zhang, S., Cohen, Y., Chen, W., Yamada, M., Hamaguchi, T., Sanjo, N., Mizusawa, H., Nakamura, Y., Kitamoto, T., Collins, S.J., Boyd, A., Will, R.G., Knight, R., Ponto, C., Zerr, I., Kraus, T.F.J., Eigenbrod, S., Giese, A., Calero, M., de Pedro-Cuesta, J., Haïk, S., Laplanche, J.-L., Bouaziz-Amar, E., Brandel, J.-P., Capellari, S., Parchi, P., Poleggi, A., Ladogana, A., O’Donnell-Luria, A.H., Karczewski, K.J., Marshall, J.L., Boehnke, M., Laakso, M., Mohlke, K.L., Kähler, A., Chambert, K., McCarroll, S., Sullivan, P.F., Hultman, C.M., Purcell, S.M., Sklar, P., van der Lee, S.J., Rozemuller, A., Jansen, C., Hofman, A., Kraaij, R., van Rooij, J.G.J., Ikram, M.A., Uitterlinden, A.G., van Duijn, C.M., Exome Aggregation Consortium (ExAC), Daly, M.J., MacArthur, D.G., 2016. Quantifying prion disease penetrance using large population control cohorts. Sci. Transl. Med. 8, 322ra9. doi:10.1126/scitranslmed.aad5169
10.	Nielsen, N.M., Frisch, M., Rostgaard, K., Wohlfahrt, J., Hjalgrim, H., Koch-Henriksen, N., Melbye, M., Westergaard, T., 2008. Autoimmune diseases in patients with multiple sclerosis and their first-degree relatives: a nationwide cohort study in Denmark. Mult. Scler. Houndmills Basingstoke Engl. 14, 823–829. doi:10.1177/1352458508088936
11.	Ramagopalan, S.V., Dyment, D.A., Valdar, W., Herrera, B.M., Criscuoli, M., Yee, I.M.L., Sadovnick, A.D., Ebers, G.C., Canadian Collaborative Study Group, 2007. Autoimmune disease in families with multiple sclerosis: a population-based study. Lancet Neurol. 6, 604–610. doi:10.1016/S1474-4422(07)70132-1
12.	Richards, S., Aziz, N., Bale, S., Bick, D., Das, S., Gastier-Foster, J., Grody, W.W., Hegde, M., Lyon, E., Spector, E., Voelkerding, K., Rehm, H.L., ACMG Laboratory Quality Assurance Committee, 2015. Standards and guidelines for the interpretation of sequence variants: a joint consensus recommendation of the American College of Medical Genetics and Genomics and the Association for Molecular Pathology. Genet. Med. Off. J. Am. Coll. Med. Genet. 17, 405–424. doi:10.1038/gim.2015.30
13.	Rodan, L.H., Cohen, J., Fatemi, A., Gillis, T., Lucente, D., Gusella, J., Picker, J.D., 2016. A novel neurodevelopmental disorder associated with compound heterozygous variants in the huntingtin gene. Eur. J. Hum. Genet. EJHG. doi:10.1038/ejhg.2016.74
14.	Sadovnick, A.D., Paty, D.W., Yannakoulias, G., 1989. Concurrence of multiple sclerosis and inflammatory bowel disease. N. Engl. J. Med. 321, 762–763.
15.	Sadovnick, A.D., Risch, N.J., Ebers, G.C., 1998. Canadian collaborative project on genetic susceptibility to MS, phase 2: rationale and method. Canadian Collaborative Study Group. Can. J. Neurol. Sci. J. Can. Sci. Neurol. 25, 216–221.
16.	Singh, T., Kurki, M.I., Curtis, D., Purcell, S.M., Crooks, L., McRae, J., Suvisaari, J., Chheda, H., Blackwood, D., Breen, G., Pietiläinen, O., Gerety, S.S., Ayub, M., Blyth, M., Cole, T., Collier, D., Coomber, E.L., Craddock, N., Daly, M.J., Danesh, J., DiForti, M., Foster, A., Freimer, N.B., Geschwind, D., Johnstone, M., Joss, S., Kirov, G., Körkkö, J., Kuismin, O., Holmans, P., Hultman, C.M., Iyegbe, C., Lönnqvist, J., Männikkö, M., McCarroll, S.A., McGuffin, P., McIntosh, A.M., McQuillin, A., Moilanen, J.S., Moore, C., Murray, R.M., Newbury-Ecob, R., Ouwehand, W., Paunio, T., Prigmore, E., Rees, E., Roberts, D., Sambrook, J., Sklar, P., Clair, D.S., Veijola, J., Walters, J.T.R., Williams, H., Swedish Schizophrenia Study, INTERVAL Study, DDD Study, UK10 K Consortium, Sullivan, P.F., Hurles, M.E., O’Donovan, M.C., Palotie, A., Owen, M.J., Barrett, J.C., 2016. Rare loss-of-function variants in SETD1A are associated with schizophrenia and developmental disorders. Nat. Neurosci. 19, 571–577. doi:10.1038/nn.4267
17.	Traboulsee, A.L., Bernales, C.Q., Ross, J.P., Lee, J.D., Sadovnick, A.D., Vilariño-Güell, C., 2014. Genetic variants in IL2RA and IL7R affect multiple sclerosis disease risk and progression. Neurogenetics 15, 165–169. doi:10.1007/s10048-014-0403-3
18.	Walsh, R., 2016. Reassessment of Mendelian gene pathogenicity using 7,855 cardiomyopathy cases and 60,706 reference samples. Genet. Med. In press. doi:doi:10.1038/gim.2016.90
19.	Wang, Z., Sadovnick, A.D., Traboulsee, A.L., Ross, J.P., Bernales, C.Q., Encarnacion, M., Yee, I.M., de Lemos, M., Greenwood, T., Lee, J.D., Wright, G., Ross, C.J., Zhang, S., Song, W., Vilariño-Güell, C., 2016. Nuclear Receptor NR1H3 in Familial Multiple Sclerosis. Neuron 90, 948–954. doi:10.1016/j.neuron.2016.04.039
20.	Watts, J.C., Giles, K., Serban, A., Patel, S., Oehler, A., Bhardwaj, S., Guan, S., Greicius, M.D., Miller, B.L., DeArmond, S.J., Geschwind, M.D., Prusiner, S.B., 2015. Modulation of Creutzfeldt-Jakob disease prion propagation by the A224V mutation. Ann. Neurol. 78, 540–553. doi:10.1002/ana.24463
21.	Weren, R.D.A., Ligtenberg, M.J.L., Kets, C.M., de Voer, R.M., Verwiel, E.T.P., Spruijt, L., van Zelst-Stams, W.A.G., Jongmans, M.C., Gilissen, C., Hehir-Kwa, J.Y., Hoischen, A., Shendure, J., Boyle, E.A., Kamping, E.J., Nagtegaal, I.D., Tops, B.B.J., Nagengast, F.M., Geurts van Kessel, A., van Krieken, J.H.J.M., Kuiper, R.P., Hoogerbrugge, N., 2015. A germline homozygous mutation in the base-excision repair gene NTHL1 causes adenomatous polyposis and colorectal cancer. Nat. Genet. 47, 668–671. doi:10.1038/ng.3287



